pkg/versioning/versioning.go

```diff
@@ -13,13 +13,13 @@ import (
 
 // Coordinates to address the artifact coordinates like groupId, artifactId, version and packaging
 type Coordinates struct {
-	GroupID    string
-	ArtifactID string
-	Version    string
-	Packaging  string
-	BuildPath  string
-	URL        string
-	PURL       string
+	GroupID    string `json:"groupId"`
+	ArtifactID string `json:"artifactId"`
+	Version    string `json:"version"`
+	Packaging  string `json:"packaging"`
+	BuildPath  string `json:"buildPath"`
+	URL        string `json:"url"`
+	PURL       string `json:"purl"`
```

Comment:
What does this stand for?

Reply:
This is required for an pub sub event and I don't want to duplicate the struct